static/css/tools/choropleth.scss

```diff
@@ -14,8 +14,10 @@
  * limitations under the License.
  */
 
- @import '../tools/base_tools';
- @import '../draw';
+@import '../tools/base_tools';
+@import '../tools/statvar_menu';
+@import '../timeline';
```

Comment:
Do you need all the css from timeline? There shouldn't be any... if there are shared items, they should be moved out of timeline into base_tools or statvar_menu.

Reply:
Copied over the one property that was needed